src/Microsoft.AspNetCore.Mvc.Razor/TagHelpers/TagHelperComponentManager.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+
+namespace Microsoft.AspNetCore.Mvc.Razor.TagHelpers
```

Comment:
This should be in a more secretive namespace like `.Internal`, there's no need for anyone to interact with this type.